docs/getting_started/quickstart_guide.md

```diff
@@ -17,6 +17,12 @@ next-page-title: Getting started with workflow development
 
 In this guide, you will create and run a Flyte workflow in a local Python environment to generate the output "Hello, world!"
 
+```{note}
```

Comment:
Let's use the original formatting used from before:

~~~
````{dropdown} Want to try Flyte on the browser?
:title: text-muted
:animate: fade-in-slide-down

```{link-button} https://sandbox.union.ai/
---
classes: try-hosted-flyte btn-warning btn-block
text: Hosted Flyte Sandbox
---
```
```{div} text-muted
*Courtesy of [Union.ai](https://www.union.ai/)*
```
````
~~~